src/libraries/System.Security.Cryptography/src/Resources/Strings.resx

```diff
@@ -933,4 +993,13 @@
   <data name="Cryptography_X509_PfxWithoutPassword_ProblemFound" xml:space="preserve">
     <value>There was a problem with the PKCS12 (PFX) without a supplied password. See https://go.microsoft.com/fwlink/?linkid=2233907 for more information.</value>
   </data>
-</root>
+  <data name="Argument_PublicKeyWrongSizeForAlgorithm" xml:space="preserve">
+    <value>The public key is not the correct size for the indicated algorithm.</value>
+  </data>
+  <data name="Argument_SecretKeyWrongSizeForAlgorithm" xml:space="preserve">
+    <value>The secret key is not the correct size for the indicated algorithm.</value>
+  </data>
+  <data name="Argument_PrivateSeedWrongSizeForAlgorithm" xml:space="preserve">
+    <value>The private seed is not the correct size for the indicated algorithm.</value>
+  </data>
```

Comment:
I think the entries in this file are at least vaguely alphabetical.  The ordering should be maintained.